django_stomp/debug_callback_view/views.py

```diff
@@ -0,0 +1,65 @@
+import importlib
+import json
+import logging
+
+from django.http import HttpRequest
+from django.http import JsonResponse
+
+from django_stomp.services.consumer import Payload
+
+_logger = logging.getLogger(__name__)
+
+
+def mock_ack():
+    return
+
+
+def mock_nack():
+    return
+
+
+def debug_callback_view(request: HttpRequest):
+    """
+    View wrapper to call pubsub logic
+
+    Request Body:
+    ```
+        {
+            "callback_function_path": "path.to.your.function",
+            "payload_body": {
+                "user_id_ref": "0c2b3b31-6b98-426b-93d1-fb4b97038e23"
+            },
+            "payload_headers": {
+                "fake": "headers"
+            }
+        }
+    ```
+    """
+    if not request.method == "POST":
+        return JsonResponse({"status": "error", "detail": "Only POST method is allowed"})
```

Comment:
**suggestion:** I'd change the status code for this return (and other error cases) to not return 200. 

Since the request failed, IMO it doesn't make sense to return a 200 with status `error` or `warning`.